pkg/validator/pkl.go

```diff
@@ -0,0 +1,32 @@
+package validator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/apple/pkl-go/pkl"
+)
+
+// PklValidator is used to validate a byte slice that is intended to represent a
+// PKL file.
+type PklValidator struct{}
+
+// Validate attempts to evaluate the provided byte slice as a PKL file.
+func (PklValidator) Validate(b []byte) (bool, error) {
+	ctx := context.Background()
+
+	// Convert the byte slice to a ModuleSource using TextSource
+	source := pkl.TextSource(string(b))
```

Comment:
Are there conditions where `pkl.TextSource` will return an err? Do we need to handle the err?